README.md

```diff
@@ -1,2 +1,32 @@
 # Portfolio
-my Portfolio
+
+# 
+> this project will display "my postpolio"
```

Comment:
- [ ] As indicated by the previous reviewer, kindly correct the typo here to enhance the professional look of the Readme. `my postpolio` to `my portfolio`